packages/headless/src/features/attached-results/attached-results-state.ts

```diff
@@ -0,0 +1,24 @@
+export interface AttachedResult {
+  articleLanguage?: string;
```

Comment:
Are these values all stored in the salesforce CoveoAttachedResult record?

Reply:
Yup,
That's the question I asked at a scrum, either we keep the bare minimum in headless state or we just add those values here but make them optional pretty much.

Reply:
If the values would be used in a future quantic-based attachedResults component then It's probably good to keep them here so the component doesn't need to fetch the data anywhere else.